cmd/docker-app/bundle_test.go

```diff
@@ -24,30 +23,27 @@ func TestMakeInvocationImage(t *testing.T) {
 		{
 			name:      "specify-image-name-and-namespace",
 			imageName: "my-invocation-image",
-			namespace: "my-namespace",
-			expected:  "my-namespace/my-invocation-image",
+			expected:  "my-invocation-image",
 		},
 		{
 			name:     "simple-metadata",
 			meta:     metadata.AppMetadata{Name: "name", Version: "version"},
 			expected: "name:version-invoc",
 		},
 		{
-			name:      "simple-metadata-with-overridden-namespace",
-			namespace: "my-namespace",
-			meta:      metadata.AppMetadata{Name: "name", Version: "version"},
-			expected:  "my-namespace/name:version-invoc",
+			name:     "simple-metadata-with-overridden-namespace",
```

Comment:
Can you check the relevance of these test cases? I don't think a test with `overridden-namespace` in its name is needed anymore 🤔

Reply:
Done

Reply:
So why are these tests still here? 😅

Reply:
Ops... My bad! Corrected